views/allTags.ejs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Michael's OMDB Movie Search</title>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+	<!-- Latest compiled and minified CSS -- -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+	<!-- <!-- Latest compiled and minified JavaScript -- -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="/static/css/style.css">
+</head>
+<body>
+
+<div class="container">
+<h1><center>All Tags</center></h1>
+<% alltags.forEach(function(tags) { %>
```

Comment:
Let's design this so that it also lists the movies associated with the tag in question. Note that you'll need to include favorites when doing the db lookup, which I'll show you how to do in the controller.
```
<% alltags.forEach(function(tag) { %>
	<ul>
		<li><a href="/favorites/tags/<%= tag.id %>"><%= tag.tag %></a></li>
		<% tag.favorites.forEach(function(favorite){ %>
			<p><%= favorite.title %></p>
		<% }); %>
	</ul>
<% }); %>
```